src/config/network/wpa_supplicant.md

```diff
@@ -3,7 +3,29 @@
 The `wpa_supplicant` package is installed by default on the base system. It
 includes utilities to configure wireless interfaces and handle wireless security
 protocols. To use wpa_supplicant, you will need to enable [the wpa_supplicant
-service](#the-wpa_supplicant-service).
+service](../services/index.md#enabling-services). 
+
+Then you need to run `rfkill` to make sure your radio device is not blocked.
+
+Run `rfkill unblock id|type` if it is shown blocked.
+
+## Using wpa_cli
+
+`wpa_cli` is a tool comes with base system. You can use `wpa_cli` to manage `wpa_supplicant` from the command line, be sure
+to specify which network interface to use via the `-i` option. Not doing so can result in various `wpa_cli` commands (for example, `scan` and
+`scan_results`) not producing the expected output.
+
+Sample Setup Commands:
```

Comment:
"An example of the procedure to connect to an access point called `wifi_name` can be seen below:"